.github/workflows/jit.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           - target: aarch64-apple-darwin/clang
             architecture: aarch64
             runner: macos-14
-          - target: x86_64-unknown-linux-gnu/gcc
+          - target: x86_64-pc-linux-gnu/gcc
```

Comment:
I discovered that this is actually what the runner is when I was working on #129331